pkg/websocket/websocket.go

```diff
@@ -0,0 +1,64 @@
+package websocket
```

Comment:
Nice idea to extract this to package. Maybe some different name, like `wslistener`, or `wsstream` if we will add more streaming related functionalities.